apps/test/unit/templates/certificates/CongratsTest.js

```diff
@@ -61,4 +62,17 @@ describe('Congrats', () => {
     );
     expect(wrapper.find(TeachersBeyondHoc).exists()).to.be.false;
   });
+
+  it('renders a PetitionCallToAction component, regardless of user type', () => {
```

Comment:
I think this would be more accurate:
```suggestion
  it('renders a PetitionCallToAction component for signed out user', () => {
```

Reply:
Oh, I was under the impression that everyone would see these. Is it only if someone is signed out?

Reply:
yes, everyone should see these, but your test case appears to only be testing that signed out users will see them.

Reply:
Ah, I see. I'm following the pattern here:
https://github.com/code-dot-org/code-dot-org/blob/staging/apps/test/unit/templates/certificates/CongratsTest.js#L21-L30

Would you suggest I test all possible `userTypes`?

Reply:
yes... I wasn't thinking that initially, but I suppose testing all user types (student, teacher, signed out) would be best.

Reply:
Sounds good, I'll do that for both of these